### PR TITLE
Fixed formatting that displayed off-screen

### PR DESCRIPTION
**Pull request by hanserasmus**
The changes did not fit into the window when viewing the docs for Ubuntu install. Fixed it according to what I thought would be displayed better.

Previous version:

image

According to the "Preview tab" whilst editing it, it should be OK now.

### CODE REVIEW

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -190,10 +190,11 @@ terminal::
 * This installs the packages for the Nextcloud core system.
   ``libapache2-mod-php7.2`` provides the following PHP extensions: ``bcmath bz2
   calendar Core ctype date dba dom ereg exif fileinfo filter ftp gettext hash
-  iconv libxml mhash openssl pcre Phar posix Reflection session shmop
-  SimpleXML soap sockets SPL standard sysvmsg sysvsem sysvshm tokenizer wddx
-  xmlreader xmlwriter zlib``. If you are planning
-  on running additional apps, keep in mind that they might require additional
+  iconv libxml 
+  mhash openssl pcre Phar posix Reflection session shmop SimpleXML soap sockets
+  SPL standard sysvmsg sysvsem sysvshm tokenizer wddx xmlreader xmlwriter zlib``.
```
Comment (MorrisJobke) [inline admin_manual/installation/source_installation.rst]: Could you simply make it a code block, by this::

```rst
... following PHP extensions::

   PUT HERE THE LONG LIST
```

**Comment by hanserasmus** [comment]
@MorrisJobke as requested per comment,  think it is fixed now.

**Pull request merged**